site/styles/_buttons.scss

```diff
@@ -218,3 +218,58 @@ s72-userwishlist-button {
     /* stylelint-enable selector-max-compound-selectors, max-nesting-depth */
   }
 }
+
+.s72-btn-sso {
+  width: 100%;
+}
+
+.s72-btn-sso-veriplay {
```

Comment:
This could be a mixin that takes colour variable args so we don't have this bootstrap override stuff duplicated in two files.